src/components/Article.jsx

```diff
@@ -0,0 +1,17 @@
+import moment from "moment";
+import { formatCommentCount } from "../utils/display";
+
+export const Article = ({ article: { title, topic, author, body, comment_count, created_at }, authorName }) => (
+    <div className="Article">
```

Comment:
Watch out for div soup, you don't want to just throw everything into divs, utilise sematic html to improve SEO and accessibility